tst/EnergyPlus/unit/HeatRecovery.unit.cc

```diff
@@ -3891,7 +3891,7 @@ TEST_F(EnergyPlusFixture, HeatRecoveryHXOnMainBranch_SimHeatRecoveryTest)
     ASSERT_NEAR(-17.300, state->dataHeatRecovery->ExchCond(1).SupInTemp, 0.001); // Heat Recovery Exchanger Primary Air Inlet Temp
     ASSERT_GT(state->dataHeatRecovery->ExchCond(1).SupOutTemp,
               state->dataHeatRecovery->ExchCond(1).SupInTemp);                  // Heat Recovery Exchanger is On in heating mode
-    ASSERT_NEAR(23.000, state->dataHeatRecovery->ExchCond(1).SecInTemp, 0.001); // Heat Recovery Exchanger Secondary Air Inlet Temp
+    ASSERT_NEAR(22.947, state->dataHeatRecovery->ExchCond(1).SecInTemp, 0.001); // Heat Recovery Exchanger Secondary Air Inlet Temp
```

Comment:
The heat recovery HX secondary air outlet temperature changed due to changes in the primary air mass flow rate. The increase in the primary air will lead to a decrease in the secondary air exist temperature.